### DISCUSSION TITLE
auto straight line

### DISCUSSION THREAD

**Discussion by xkmanlordx** [flxzt/rnote#1167] [Ideas]
In OneNote, when drawing a line free hand, that is _close enough_ to being straight, the software will recognize this and transform it into a proper straight line, if holing down a second at the end of the line. This is really useful when taking notes including large fractions, graphs etc. 
I know there is a shape tool to draw lines, but this would be a nice convenience to have.
Grateful for the software either way, thank you for implementing my window resizing complaint in version 0.11 <3

**Comment by Doublonmousse** [comment]
This is basically #384.

When implemented, it could be extended to a few other shapes as well but we first need to implement a way to detect holding down the pen in the same position.
